lightdm/xfce_config.conf

```diff
@@ -167,5 +168,6 @@ autologin-user-timeout=0
 #depth=8
 
 [SeatDefaults]
```

Comment:
This entire section can be dropped as `[Seat:*]` became the default a long time ago.